examples/profile.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE>
+<html ng-app='app'>
+  <head>
+    <title>Interaction button group example</title>
+    <meta charset="utf-8">
+    <meta name="viewport"
+          content="initial-scale=1.0, user-scalable=no, width=device-width">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <link rel="stylesheet" href="../node_modules/openlayers/css/ol.css" type="text/css">
+    <link rel="stylesheet" href="http://getbootstrap.com/dist/css/bootstrap.min.css" type="text/css">
+    <style>
+
+        .profile {
+          width: 450px;
+          height: 250px;
+        }
+
+    </style>
+  </head>
+  <body ng-controller="MainController as ctrl">
+    <div class="profile" ngeo-profile="ctrl.profileData"
```

Comment:
I don't think the examples should show bad practices. If the data does not change then one-time binding should be used.